examples/functions/golang/pool/mmf/matchfunction.go

```diff
@@ -66,8 +66,8 @@ func MakeMatches(params *mmfHarness.MatchFunctionParams) ([]*pb.Match, error) {
 func scoreCalculator(tickets []*pb.Ticket) float64 {
 	matchScore := 0.0
 	for _, ticket := range tickets {
-		for _, v := range ticket.GetProperties().GetFields() {
-			matchScore += v.GetNumberValue()
+		for _, v := range ticket.GetSearchFields().GetDoubleArgs() {
```

Comment:
Some comment on what this does? :) I know you stumbled on what the previous one was doing - so even if not scientific .. a note stating this is what is does as a placeholder would be great.

Reply:
There was already one, expanded a little.